canonical-websites/www.ubuntu.com
# Clean-up and rationalisation of legal section

## Done

* update [/legal/terms-and-policies/thank-you](http://0.0.0.0:8001/legal/terms-and-policies/thank-you) title and created copy doc
* Updated titles on Japanese language pages
* Removed unused /lega/bootstack pages
* moved /contributors-agreement into /contributors
* added an older/latest version history to /legal/developer-terms-and-conditions pages
* flattened ipp archive file - 2013-05-14.html from folder
* renamed managed-services-services-description to managed-services-description

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- See the **Done** section and click around
- Look at the [legal folder on gdocs](https://drive.google.com/drive/folders/0B3hNyKyLEnE6clUxQndBME5uRXM)


## Issue / Card

Fixes https://github.com/ubuntudesign/web-squad/issues/873
